client/state/reader/follows/actions.js

```diff
@@ -9,6 +9,8 @@ import debugModule from 'debug';
 import {
 	READER_FOLLOW,
 	READER_UNFOLLOW,
+	READER_RECORD_FOLLOW,
```

Comment:
they're used in the current flux bound follow button and were used to drive cold start. I'd rather not drop them here, but in the PR where we actually switch from the flux bound follow button to the redux one.